draft-song-lake-ra.md

```diff
@@ -98,277 +99,97 @@ How the components are separated between the secure and non-secure worlds on a d
-|          |               |           | request      |           |  Attestation  |          |
-|          |   Evidence    |           |              |           |  result       |          |
-|          |<--------------+           |<-------------+           |               |          |
-|          +-------------->|           | Evidence     |           |               |          |
-|          |  Attestation  |           |              |           |               |          |
-+----------+  result       +-----------+              +-----------+               +----------+
-                                       <-------------->
-                                        EDHOC session
-~~~~~~~~~~~
-{: #fig-mutual-attestation-BB title="Overview of mutual attestation of Background-check model in both sides. EDHOC is used between A and RP." artwork-align="center"}
+## Attestation Models {#models}
 
-### Background-check model -- Passport model {#mutual-attestation-BP}
+This specification supports both the RATS background-check model (see in {{bg}}) and passport model (see in{{pp}}).
+The corresponding EAD items for background-check model and the passport model are independent between each other.
+The detailed EAD items definitions are in {{ead-bg}} for the background-check model and {{ead-pp}} for the passport model.
 
-This use case is applicable when the EDHOC Initiator is constrained in terms of connectivity.
-In this case, the mutual attestation begins with a forward background-check attestation, followed by a reverse passport attestation.
+### Background-check Model (BG) {#bg}
 
-1. EDHOC Initiator sends the first EDHOC message with an Attestation proposal in EAD_1.
+In background-check model, the Attester sends the evidence to the Relying Party.
+The Relying Party transfers the evidence to the Verifier and gets back the attestation result from the Verifier.
 
-2. In EAD_2, two EAD items are included: an Attestation request responding to the EDHOC Initiator's proposal and a Result proposal to initiate a reverse passport attestation.
-The EDHOC Responder SHOULD connect to a Verifier to select the evidence type for the forward attestation in background-check model.
+An EDHOC session is established between the Attester and the Relying Party.
+A negotiation of the evidence type is required before the Attester sends the evidence.
+All three parties must agree on a selected evidence type.
 
-3. In EAD_3, two EAD items are included: an Evidence and a Result request.
-The EDHOC Responder SHOULD connect to the Verfier in the passport model(VP) indicated in the Result request to obtain the Result from that Verifier.
-Simultaneously, the EDHOC Responder sends the received Evidence to the Verifier in the background-check model(VB) to get the attestation result of the first forward attestation.
+The Attester first sends a list of the provided evidence types, formatted as an Attestation proposal in an EDHOC EAD item, to the Relying Party.
```

Comment:
I _think_ it is preferred to write it like this:

The Attester first sends a list of the provided evidence types to the Relying Party.
The list is formatted as an Attestation_proposal in an EDHOC EAD item.